impala/util.py

```diff
@@ -64,6 +64,18 @@ def as_pandas(cursor, coerce_float=False):
                                   coerce_float=coerce_float)
 
 
+def as_dict(cursor):
```

Comment:
I think as_dicts() would be a clearer name (it sounds like it returns a single dictionary).

Reply:
You are right! Another option is `dictfetchall`. This name refers to the fetchall operation that occurs on the function.

Reply:
Either is fine by me. It looks like the dictfetchall is used by django so there's some precedent.